packages/web-react/src/components/Link/demo/index.tsx

```diff
@@ -0,0 +1,30 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
```

Comment:
I am not quite sure if these ESlint disables are necessary anymore....

Reply:
removed those comments